### PR TITLE
Added Normal Gem Recipes

### PR DESCRIPTION
**Pull request by IronMaXxX**
Updated code to enable cubing normal gems to flawless gems

### CODE REVIEW

File: internal/action/cube_recipes.go [hunk 1]
```diff
@@ -20,7 +20,37 @@ type CubeRecipe struct {
 var (
 	Recipes = []CubeRecipe{
 
-		// Perfects
+		// Flawless Gems
+		{
+			Name:  "Amethyst",
```
Comment (artosimonyan) [inline internal/action/cube_recipes.go:25]: The recipe name here needs to match with the recipe.go in the config. (this is for all newly added recipes). Look how the other recipes are added.

File: internal/config/recipes.go [hunk 2]
```diff
@@ -2,6 +2,13 @@ package config
 
 var AvailableRecipes = []string{
 	// List out all the recipe names from cube_recipes.go
+	"Flawless Amethyst",
```
Comment (artosimonyan) [inline internal/config/recipes.go:5]: These recipes aren't defined.

File: internal/action/cube_recipes.go (hunk 1 above)
Comment (IronMaXxX) [reply internal/action/cube_recipes.go]: Thanks for the hint, addes Flawless to the recipes

File: internal/config/recipes.go (hunk 2 above)
Comment (IronMaXxX) [reply internal/config/recipes.go:5]: fixed recipe names in cube_recipes.go, now they are defined.

**Pull request closed without merging**